### PR TITLE
MM-1777 Formatted comments' time in RHS similar to root

### PR DESCRIPTION
**Pull request by nickago**

### CODE REVIEW

File: web/react/components/post_right.jsx [hunk 1]
```diff
@@ -146,7 +146,7 @@ CommentPost = React.createClass({
                 <div className="post__content">
                     <ul className="post-header">
                         <li className="post-header-col"><strong><UserProfile userId={post.user_id} /></strong></li>
-                        <li className="post-header-col"><time className="post-right-comment-time">{ utils.displayDateTime(post.create_at) }</time></li>
+                            <li className="post-header-col"><time className="post-right-comment-time">{ utils.displayDate(post.create_at)+' '+utils.displayTime(post.create_at)  }</time></li>
```
Comment (rgarmsen2295) [inline web/react/components/post_right.jsx]: Shouldn't these be single quotes for the style guide?

**Comment by nickago** [comment]
@rgarmsen2295 Alright, added a refactoring commit

**Comment by rgarmsen2295** [comment]
+1, looks good

**Comment by nickago** [comment]
displayDateTime is actually being used for a different kind of formatting, but I agree that we should encapsulate this in a function, commit on it's way

**Comment by nickago** [comment]
@crspeller encapsulation completed

**Comment by rgarmsen2295** [comment]
Travis issue looks like gremlins, so +1 from me on the function.

**Comment by jwilander** [comment]
@crspeller please give your consent when you get a chance and merge

**Comment by crspeller** [comment]
Looks good to me. Ready to merge once the travis build finishes.

**Comment by jwilander** [comment]
This needs a rebase.

**Pull request merged**